src/framework/components/light/system.js

```diff
@@ -67,6 +67,11 @@ pc.extend(pc, function () {
             LightComponentSystem._super.initializeComponentData.call(this, component, data, _props);
         },
 
+        removeComponent: function (entity) {
```

Comment:
Note: most newer components subscribe to 'beforeremove' or 'remove' events and handle this there. e.g. See ElementComponentSystem

If you want to override removeComponent, you need to make sure you call the parent class removeComponent method too. e.g. See ModelComponentSystem

Also note: LightComponentSystem is subscribing to 'remove' event already, but doesn't seem to have implemented the method.

Reply:
Fixed! 😀